lib/pbench/test/unit/server/test_requests.py

```diff
@@ -96,6 +95,18 @@ def test_missing_authorization_header(client, caplog, server_config):
             headers={"filename": "f.tar.xz"},
         )
         assert response.status_code == 401
+        for record in caplog.records:
+            assert record.levelname not in ("ERROR", "CRITICAL")
+
+    @staticmethod
+    def test_malformed_authorization_header(client, caplog, server_config):
```

Comment:
This is really "bad token" not "malformed"... the header format is correct. Though a "malformed" test, e.g., `Authorization: Bear cub` might not be a bad idea. too.

Reply:
My nit is bigger than your nit? Or something like that? Or whatever...

I suppose the point is that the name refers to the authorization header, not the token, while the test has a bad token but (from strict HTTP protocol point of view) the header is correctly formatted. Both cases are worth testing. Neither case is specific to the PUT API.

Reply:
@dbutenhof you mean testing with header other than a `Bearer` protocol, yeah these cases are tested under `test_user_auth.py` file

Reply:
Tracked in #2246.